### PR TITLE
SKYNET-2281: Add helm chart and ncbuild

### PR DESCRIPTION
**Pull request by nightfury1204**
Tested build job: https://jenkins.newscred.com/job/newscred-grafana-build/2/console

### CODE REVIEW

File: charts/grafana/README.md [hunk 1]
```diff
@@ -0,0 +1,523 @@
+# Grafana Helm Chart
+
+* Installs the web dashboarding system [Grafana](http://grafana.org/)
+
+## Get Repo Info
+
+```console
+helm repo add grafana https://grafana.github.io/helm-charts
```
Comment (mohsiur) [inline charts/grafana/README.md:8]: Can we update this to be consistent with newscred-grafana instead

File: ncbuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -xeuo pipefail
+
+ecr_url=304160530156.dkr.ecr.us-east-1.amazonaws.com
```
Comment (mohsiur) [inline ncbuild:4]: This is not being used remove it?

File: charts/grafana/README.md (hunk 1 above)
Comment (nightfury1204) [reply charts/grafana/README.md]: updated

File: ncbuild (hunk 2 above)
Comment (nightfury1204) [reply ncbuild]: addressed

**Pull request merged**